roles/base/tasks/redhat_tasks.yml

```diff
@@ -22,3 +23,17 @@
 
 - name: install and start ntp
   service: name=ntpd state=started enabled=yes
+
+- name: enable NFS server
+  service:
+    name: nfs-server
+    state: started
+    enabled: yes
+  when: use_nfs
```

Comment:
We can have a more general variable like say `storage-type` with possible values as `ceph`, `nfs` or `ceph,nfs`and so on which will let us install either nfs or ceph or both based on user's deployment.